src/coreclr/tools/dotnet-pgo/MethodMemoryMap.cs

```diff
@@ -47,13 +71,16 @@ internal class MethodMemoryMap
 
                 if (method != null)
                 {
-                    infos.Add(new MemoryRegionInfo
+                    JittedID jittedID = new JittedID(e.MethodID, 0);
+                    if (!info.ContainsKey(jittedID))
```

Comment:
Do we see duplicate keys in practice here? Do we know why?

Reply:
In theory if the trace contains both MethodLoad and MethodLoadVerbose events, there can be a mess here. In practice though, I haven't seen it.